src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -1,350 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Reviewers: BridgeClientCertificateManager was renamed to CertificateManager and then modified.  I'll note what is new in CertificateManager and what was not retained from BridgeClientCertificateManager.